test/MochaSpecRunner.html

```diff
@@ -0,0 +1,51 @@
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Mocha Tests</title>
+    <script type="text/javascript" src="lib/webcomponents.js"></script>
+    <link href="lib/mocha.css" rel="stylesheet" />
+    <link rel="import" href="../lab/polymer/components/my-menu-button.html">
+</head>
+
+<body>
+    <my-menu-button style="position: absolute; top: 0; right: 0" href="subpage.html">Distributed <strong>Content</strong></my-menu-button>
```

Comment:
@warpech here is the explanation: https://github.com/Palindrom/Palindrom/blob/887e2084bd76971a197e2132407744efe136f85e/dist/palindrom-dom.js#L2135 

When OT is off, `NoQueue` is used, and `NoQueue` resets by apply a patch to the root object.